index.html

```diff
@@ -125,5 +125,190 @@ <h2>I'am a Software Developer</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+              has been the industry's standard
+            </p>
+            <div class="inner-button">
+              <button type="button">html</button>
+              <button type="button">bootstrap</button>
+              <button type="button">Ruby</button>
+            </div>
+          </div>
+          <button type="button" class="inner-work-btn">see projects</button>
+        </div>
+
+        <div class="inner-work">
+          <div class="inner-work-text">
+            <h3>professional art <br />printing data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+              has been the industry's standard
+            </p>
+            <div class="inner-button">
+              <button type="button">html</button>
+              <button type="button">bootstrap</button>
+              <button type="button">Ruby</button>
+            </div>
+          </div>
+          <button type="button" class="inner-work-btn">see projects</button>
+        </div>
+
+        <div class="inner-work">
+          <div class="inner-work-text">
+            <h3>professional art <br />printing data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+              has been the industry's standard
+            </p>
+            <div class="inner-button">
+              <button type="button">html</button>
+              <button type="button">bootstrap</button>
+              <button type="button">Ruby</button>
+            </div>
+          </div>
+          <button type="button" class="inner-work-btn">see projects</button>
+        </div>
+
+        <div class="inner-work">
+          <div class="inner-work-text">
+            <h3>professional art <br />printing data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+              has been the industry's standard
+            </p>
+            <div class="inner-button">
+              <button type="button">html</button>
+              <button type="button">bootstrap</button>
+              <button type="button">Ruby</button>
+            </div>
+          </div>
+          <button type="button" class="inner-work-btn">see projects</button>
+        </div>
+
+        <div class="inner-work">
+          <div class="inner-work-text">
+            <h3>professional art <br />printing data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+              has been the industry's standard
+            </p>
+            <div class="inner-button">
+              <button type="button">html</button>
+              <button type="button">bootstrap</button>
+              <button type="button">Ruby</button>
```

Comment:
These elements don't have to be buttons since they are not intended to do anything in the project. Or is there any intension for then to have a functionality that would require the use of a button?.